src/collect-ubuntu-browser-dependencies/shared-object.ts

```diff
@@ -0,0 +1,34 @@
+import _ from "lodash";
+import calcLevenshtein from "js-levenshtein";
+import { readElf, aptFileSearch } from "./ubuntu";
+
+export const searchSharedObjectPackage = async (sharedObject: string): Promise<string> => {
+    const aptFileResult = await aptFileSearch(sharedObject);
+
+    const packages = aptFileResult.split("\n").filter(Boolean);
+
+    if (packages.includes("libc6")) {
+        return "libc6";
+    }
+
+    const relevantPackageName = _.minBy(packages, packageName => calcLevenshtein(sharedObject, packageName)) as string;
+
+    return relevantPackageName;
+};
+
+export const getBinarySharedObjectDependencies = async (binaryPath: string): Promise<string[]> => {
+    const sharedObjectRegExp = /^\s*\dx\d+\s\(NEEDED\)\s*Shared library: \[(.*)\]/gm;
```

Comment:
please, add an example for `readElf` to understand what is going on here

Reply:
In code section?
There is in test section